models/stdp_dopa_connection.h

```diff
@@ -565,8 +567,9 @@ STDPDopaConnection< targetidentifierT >::send( Event& e,
     process_dopa_spikes_( dopa_spikes, t0, start->t_ + dendritic_delay, cp );
     t0 = start->t_ + dendritic_delay;
     minus_dt = t_last_update_ - t0;
-    if ( start->t_ < t_spike ) // only depression if pre- and postsyn. spike
-                               // occur at the same time
+    // only depression if pre- and postsyn. spike occur at the same time
+    if ( t_spike - start->t_ > 1.0
+        * kernel().connection_manager.get_stdp_eps() )
```

Comment:
I would drop the multiplication with `1.0`. I also don't understand the comment: the comment talks about pre- and post-spike being at the same time, but then you facilitate only if there is a non-zero time difference. Your code does the same thing as the original code, so my confusion also applies to the comment in the original code.

Reply:
I agree and have removed the 1.0.

The comment is misleading as it's about synaptic depression. I have adjusted it such that it explains that facilitation is skipped in case of 0 temporal difference between pre- and post spike.